### PR TITLE
add game manager

### PR DESCRIPTION
**Pull request by Gr33ndev**
Closes Gamify-IT/issues#100
Closes Gamify-IT/issues#103
Closes Gamify-IT/issues#104
Closes Gamify-IT/issues#108
Closes Gamify-IT/issues#116
Closes Gamify-IT/issues#117

### CODE REVIEW

**Comment by delvh** [comment]
How much longer will this PR be WIP?
Feature freeze is tonight...

**Comment by Gr33ndev** [comment]
> 

It will be WIP until we finshed the features... The reviewers will have to hurry up to merge the branch before midnight...

**Comment by Gr33ndev** [comment]
Requires https://github.com/Gamify-IT/overworld-backend/pull/13

**Review by WuestFlorian — APPROVED** [review]

**Comment by Gr33ndev** [comment]
Please squash merge this PR

File: .github/workflows/deploy-docker-image-on-release.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - "main"
+      - "feature/game-manager"
```
Comment (marlau97) [inline .github/workflows/deploy-docker-image-on-release.yml:7]: do we still need this after merging?

**Review by marlau97 — APPROVED** [review]

**Review by cdingm — APPROVED** [review]
cant test everything since docker doesnt work on my machine, but during our discussion and testing today it looked good

File: .github/workflows/deploy-docker-image-on-release.yml (hunk 1 above)
Comment (Gr33ndev) [reply .github/workflows/deploy-docker-image-on-release.yml:7]: Will fix this tomorrow after merge

**Pull request merged**